.circleci/config.yml

```
@@ -80,7 +80,7 @@ jobs:
       - run:
           name: Stage showcase in proto dependencies
           command: |
-              git clone -b input-contract https://github.com/googleapis/api-common-protos.git
+              git clone -b input-contract-edits https://github.com/googleapis/api-common-protos.git
```

* TODO(me): Try to merge googleapis/api-common-protos#20 before this goes in.
  * TODO(landrito): Revert this ones it that is done.

```suggestion
              git clone -b input-contract https://github.com/googleapis/api-common-protos.git
```

---

Will Do!

---

Done.